pkg/planner/cascades/old/testdata/transformation_rules_suite_out.json

```diff
@@ -496,7 +496,7 @@
           "Group#6 Schema:[test.t.a,Column#25]",
           "    Projection_5 input:[Group#7], test.t.a, test.t.b->Column#25",
           "Group#7 Schema:[test.t.a,test.t.b]",
-          "    TopN_14 input:[Group#8], test.t.b, offset:0, count:1",
+          "    TopN_14 input:[Group#8], , offset:0, count:1",
```

Comment:
@Rustin170506 is this test result change correct?

Reply:
No, I think we shouldn't have any plan changes here.